godotproject/PlayerInputHandler.gd

```diff
@@ -0,0 +1,118 @@
+extends Node
```

Comment:
Was this the script used to track the inputs for making the demo video? If so can it be deleted?

Reply:
It's still in use, it's used both for recording and playback of inputs.